react/features/base/toolbox/components/ToolboxItem.web.js

```diff
@@ -27,14 +62,21 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
             elementAfter,
             onClick,
             showLabel,
-            tooltipPosition
+            tooltipPosition,
+            toggled
         } = this.props;
         const className = showLabel ? 'overflow-menu-item' : 'toolbox-button';
         const props = {
+            'aria-pressed': toggled,
+            'aria-disabled': disabled,
             'aria-label': this.accessibilityLabel,
             className: className + (disabled ? ' disabled' : ''),
-            onClick: disabled ? undefined : onClick
+            onClick: disabled ? undefined : onClick,
+            onKeyDown: this._onKeyDown,
+            tabIndex: 0,
```

Comment:
is tabIndex necessary?

Reply:
It is what puts the `li` or `div` into the tab rotation which makes it accessible by keyboard in the first place. If you were using a `button` here then it wouldn't be needed.